test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateModerationResponse.cs

```diff
@@ -44,6 +44,6 @@ internal CreateModerationResponse(string id, string model, IReadOnlyList<CreateR
         /// <summary> The model used to generate the moderation results. </summary>
         public string Model { get; }
         /// <summary> A list of moderation objects. </summary>
-        public IReadOnlyList<CreateResult> Results { get; }
+        public IReadOnlyList<CreateModerationResponseResult> Results { get; }
```

Comment:
This is a list of anonymous models: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/moderation/models.tsp#L27